### PR TITLE
Use OPA SDK

### PR DESCRIPTION
**Pull request by anderseknert**
Refactor code to use the OPA SDK

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: internal/api/api.go [hunk 1]
```diff
@@ -71,13 +73,13 @@ func New(engine *opa.OPA) *API {
 }
 
 // Run starts the HTTP server.
-func (api *API) Run(ctx context.Context) error {
+func (api *API) Run(_ context.Context) error {
```
Comment (srenatus) [inline internal/api/api.go:76]: [nit] We're the only callers. If we don't care for the context argument, we could drop it.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 /config.yaml
+/opa-conf.yaml
```
Comment (srenatus) [inline .gitignore:2]: Is this a default file name of some sort...?

**Review by srenatus — APPROVED** [review]
LGTM

File: .gitignore (hunk 2 above)
Comment (anderseknert) [reply .gitignore:2]: Heh, it's the one used by Styra DAS, which is what I used for testing.. as close to a "standard" you'll get :P

File: internal/api/api.go (hunk 1 above)
Comment (anderseknert) [reply internal/api/api.go:76]: Yeah, I basically just fixed some linter warnings along the way.. someone could definitely do some more thorough touch ups here and there.. I'll let this be for now though.

**Pull request merged**